db/migrate/20150413103214_add_solicitor_and_cco_uuids_to_defence_requests.rb

```diff
@@ -0,0 +1,6 @@
+class AddSolicitorAndCcoUuidsToDefenceRequests < ActiveRecord::Migration
+  def change
+    add_column :defence_requests, :solicitor_uuid, :uuid
+    add_column :defence_requests, :cco_uuid, :uuid
```

Comment:
Opps - I didn't notice there is already another migration to rename them to `_uid`. So ignore above.